packages/manager/apps/container/src/container/nav-reshuffle/sidebar/tooltip/SidebarLinkTooltip.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FunctionComponent } from "react";
+import { Tooltip } from "react-tooltip";
+
+type Props = { 
+    children: React.ReactNode, 
+    content: string 
+};
+
+export const ShortSidebarLinkTooltip: FunctionComponent<Props> = ({
+    content,
+    children
+}) => {
+
+    const id = 'SidebarLinkTooltip';
+
+    return (
+        <>
+            <span data-tooltip-id={id}
+                data-tooltip-position-strategy={'fixed'}
+                data-tooltip-content={content}
+                data-tooltip-place="right"
+            >
+                {children}
+            </span>
+            <Tooltip
```

Comment:
Why not using ODS tooltip ?
https://ovh.github.io/design-system/latest/?path=/story/tooltip--default

Reply:
i was expecting this question. i initially went with ODS but I switched to react tooltip because it has a hover anomaly with the navbar, it blocks the hover of the element below, we cannot manage the direction of the tooltip, and ODS does not manage well the fixed position because the sidebar is in hidden overflow.

Reply:
Then we probably need to report a bug (because it seems this should be working: https://zeroheight.com/6fc8a63f7/p/719b66-tooltip/t/page-65d34c-74844005-40a887-11)

Reply:
I created a bug ticket with the code: ODS-1367